src/main/java/uk/gov/companieshouse/ocr/api/ThreadConfig.java

```diff
@@ -54,13 +54,14 @@ public ThreadPoolTaskExecutor imageToTextTaskExecutor() {
     }
 
     @Bean (name=OCR_REQUEST_EXECUTOR_BEAN)
-    public ThreadPoolTaskExecutor taskExecutor() {
+    public ThreadPoolTaskExecutor ocrRequestTaskExecutor() {
 
         ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
 
         LOG.info("Creating a thread pool for the " + OCR_REQUEST_EXECUTOR_BEAN);
 
         executor.setThreadNamePrefix(OCR_REQUEST_THREAD_NAME_PREFIX);
+        executor.setQueueCapacity(0); // we just want one queue - the one on the Tesseract image to text queue
```

Comment:
Could we define a constant instead of hardcode "0" here ? Comment could also moved to constant if required.

Reply:
I have used a constant and added some JavaDoc to the method that has created the ThreadPoolTaskExecutor to explain why we are not created a Queue for this ThreadPoolTaskExecutor